TGM/src/main/java/com/minehut/tgm/modules/DeathModule.java

```diff
@@ -18,14 +16,16 @@
      * This is to prevent players from "combat logging"
      * by team swapping to avoid getting a death.
      */
-    @EventHandler
-    public void onTeamChange(TeamChangeEvent event) {
-        if (TGM.get().getMatchManager().getMatch().getMatchStatus() == MatchStatus.MID) {
-            if (event.getOldTeam() != null && !event.getOldTeam().isSpectator()) {
-                event.getPlayerContext().getPlayer().setHealth(0);
-            }
-        }
-    }
+
+    // not need because i prevent people changing teams during the game
+//    @EventHandler
+//    public void onTeamChange(TeamChangeEvent event) {
+//        if (TGM.get().getMatchManager().getMatch().getMatchStatus() == MatchStatus.MID) {
+//            if (event.getOldTeam() != null && !event.getOldTeam().isSpectator()) {
+//                event.getPlayerContext().getPlayer().setHealth(0);
+//            }
+//        }
+//    }
```

Comment:
You compiled your version of the code and tested it, and people still died when team switching? The only way that would be true is if this isn't the only place in the code where that's handled.